### DISCUSSION TITLE
Where to download  ImageMagick-6.8.9-1-Q16-x86-dll.exe

### DISCUSSION THREAD

**Discussion by a6965921** [ImageMagick/ImageMagick#4586] [Help]
Where to download  ImageMagick-6.8.9-1-Q16-x86-dll.exe

**Comment by dlemstra** [comment]
You should not use that version anymore and upgrade to a more recent version.

**Reply by a6965921** [reply]
> 

my php version is 5.4 
so i must use it 
![image](https://user-images.githubusercontent.com/1198840/145774399-90d4b773-47ed-4081-8c9e-b407aab6179a.png)

**Reply by dlemstra** [reply]
I would advise you to upgrade PHP or IMagick then. That version of ImageMagick is very old and we have made a lot of security patches over the years.
